advanced/advanced_numpy/examples/stride-diagonals-answer.py

```diff
@@ -5,6 +5,7 @@
 Solution showing how to use as_strided to stride in diagonal.
 
 """
+from __future__ import print_function
```

Comment:
Same remark here. That comment could mention that in Python 3, this line is not needed.